### PR TITLE
make SAML attributes more discoverable

### PR DESCRIPTION
**Pull request by StarfallProjects**

### CODE REVIEW

**Review by yob — COMMENTED** [review]
This is great!

I haven't read all the copy changes in detail, but it's definitely the case that we support those SAML user attributes from any SAML SSO provider that is willing to provide them.

I'm not sure if we've actively tested any but Okta, but the code paths are common.

File: pages/integrations/sso/google_workspace_saml.md.erb [hunk 1]
```diff
@@ -1,43 +1,45 @@
-# Single Sign-On with Google Workspace (SAML)
+# Single sign-on with Google Workspace (SAML)
 
-As an alternative to [Google Workspace SSO using OpenID](/docs/integrations/sso/g-suite), Google Workspace can be used as an SSO provider for your Buildkite organization using SAML.
+As an alternative to [Google Workspace SSO using OpenID](/docs/integrations/sso/g-suite), you can us Google Workspace as an SSO provider for your Buildkite organization using SAML.
```
Comment (plaindocs) [inline pages/integrations/sso/google_workspace_saml.md.erb:3]:
```suggestion
As an alternative to [Google Workspace SSO using OpenID](/docs/integrations/sso/g-suite), you can use Google Workspace as an SSO provider for your Buildkite organization using SAML.
```

File: pages/integrations/sso/google_workspace_saml.md.erb [hunk 2]
```diff
@@ -55,27 +57,29 @@ Click the 'Edit Settings' button, choose the Manual data option, and enter the I
...
         <td>X.509 certificate</td>
         <td>
-            The public key certificate generated for you by Google Workspace that you downloaded during the previous step. You'll need the whole file, not just a link to the file.
+            The public key certificate generated for you by Google Workspace that you downloaded during the previous step. You need the whole file, not just a link to the file.
         </td>
     </tr>
 </table>
 
-Save your new settings, and you'll be returned to your provider page.
+Save your new settings, and Buildkite returns you to your provider page.
```
Comment (plaindocs) [inline pages/integrations/sso/google_workspace_saml.md.erb:65]: Not super keen on 

> Buildkite returns you to your provider page.

How about something something confirm that provider page opens? something something returned to your provider page? it's all a bit meh

File: pages/integrations/sso/onelogin.md.erb [hunk 3]
```diff
@@ -1,25 +1,27 @@
 # Single Sign-On with OneLogin
 
-OneLogin can be used as an SSO provider for your Buildkite organization. To complete this tutorial, you will need admin privileges for both OneLogin and Buildkite.
+You can us OneLogin as an SSO provider for your Buildkite organization. To complete this tutorial, you need admin privileges for both OneLogin and Buildkite.
```
Comment (plaindocs) [inline pages/integrations/sso/onelogin.md.erb:3]:
```suggestion
You can use OneLogin as an SSO provider for your Buildkite organization. To complete this tutorial, you need admin privileges for both OneLogin and Buildkite.
```

**Review by plaindocs — APPROVED** [review]
A ton of great consistency fixes, as well as shared SAML attributes. Nice one!

File: pages/integrations/sso/google_workspace_saml.md.erb (hunk 2 above)
Comment (StarfallProjects) [reply pages/integrations/sso/google_workspace_saml.md.erb:65]: How about:

Save your settings. Buildkite returns you to your provider page.

Or

Save your settings. Your provider page opens.

?

Comment (plaindocs) [reply pages/integrations/sso/google_workspace_saml.md.erb:65]:
```suggestion
Save your settings. Your provider page opens.
```

**Pull request merged**